README.md

```diff
@@ -26,11 +26,11 @@ Notification must be sent when a new report is available.
 
 List the dependencies of the Analysis-functionality.
 
-1. Access to the Server containing the telemetrics in a csv file
-1. _enter dependency
-1. _enter dependency
+1. Access to the Server containing the telemetrics in a csv file.
+2. Mail server access for Notification.
+3. PDF report generator.
+4. Access to write/store the PDF report in server.
```

Comment:
Kindly check on point 4

Reply:
Yes, We need access for server/folder access to PDF get stored